sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample07_CrudOperations.md

```diff
@@ -63,7 +63,7 @@ await client.DeleteQueueAsync(queueName);
 ```C# Snippet:CreateTopicAndSubscription
 string connectionString = "<connection_string>";
 string topicName = "<topic_name>";
-var client = new ServiceBusManagementClient(connectionString);
+var client = new ServiceBusAdministrationClient(connectionString);
```

Comment:
We'll need to update this in the test where the snippet lives and then run the script to regen snippets.   Looks like that is [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample07_CrudOperations.cs#L126).

@JoshLove-msft:  This snippet seems to be creating a different client for the snippet then the test, which is where this discrepancy seems to have crept in.  Any reason that you can see why we can't move the client creation on L118 and L126 to a new location after the `#if` block?

Reply:
I think the reason for this is that we need the client in the finally block. We should at least fix this occurrence in the code though.

Reply:
See https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets